Makefile

```diff
@@ -71,6 +72,11 @@ css/codemirror.css:
 	cat node_modules/codemirror/lib/codemirror.css \
 	node_modules/codemirror/addon/fold/foldgutter.css > ./css/codemirror.css
 
+css/fontawesome.css:
+	cat node_modules/@fortawesome/fontawesome-free/css/fontawesome.min.css > ./css/fontawesome/css/fontawesome.min.css
+	cat node_modules/@fortawesome/fontawesome-free/css/solid.min.css > ./css/fontawesome/css/solid.min.css
+	cp -R node_modules/@fortawesome/fontawesome-free/webfonts ./css/fontawesome
```

Comment:
I should know this by now, but will the makefile always be run on a gh action or via some similar publishing step?

Reply:
The only place it is automated is in the settings for the `runonsave` vs code extension.  The built files must be committed to show up on the live site.

Reply:
Actually now that you have flagged this I am seeing that this line should not be included under `all` in the makefile, as these fontawesome dependencies are committed once and will never change until we need to upgrade the fontawesome package or bring in other parts of it.  `all` is what runs when we use the command `make` and should only include the steps that depend on source code changes.  

I amended a change just now to remove them from the `all` command.